src/js/view/controls/controlbar.js

```diff
@@ -135,11 +135,13 @@ export default class Controlbar {
             this.trigger('settingsInteraction', 'quality', true, event);
         }, localization.settings, cloneIcons('settings'));
         setAttribute(settingsButton.element(), 'aria-haspopup', 'true');
+        setAttribute(settingsButton.element(), 'aria-controls', 'jw-settings-menu');
 
         const captionsButton = button('jw-icon-cc jw-settings-submenu-button', (event) => {
             this.trigger('settingsInteraction', 'captions', false, event);
         }, localization.cc, cloneIcons('cc-off,cc-on'));
         setAttribute(captionsButton.element(), 'aria-haspopup', 'true');
```

Comment:
should we memoize the result of `.element()` here so we aren't calling it twice for both els?

Reply:
good call

Reply:
Rolled back due to failing tests.